src/structures/TextChannel.js

```diff
@@ -74,6 +74,7 @@ class TextChannel extends GuildChannel {
 
   // These are here only for documentation purposes - they are implemented by TextBasedChannel
   sendMessage() { return; }
+  sendEmbed() { return; }
```

Comment:
Need this in User and GuildMember as well - also, sort it after `sendTTSMessage`